packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -94,49 +95,84 @@ const TaxResidenceSelect = ({ field, errors, setFieldValue, values, is_changeabl
     </React.Fragment>
 );
 
-export class PersonalDetailsForm extends React.Component {
-    static contextType = PlatformContext;
+export const PersonalDetailsForm = props => {
```

Comment:
@george-usynin-deriv Isn't it better to do props destructuring here instead of doing it in a separate place? What do you think?

Reply:
that make sense 👍